### DISCUSSION TITLE
Plans to support graffle v8?

### DISCUSSION THREAD

**Discussion by onionhammer** [0no-co/gql.tada#405] [Q&A]
graphql-request, which is used heavily in the gql.data docs, has changed significantly (to my eyes), including renaming to 'graffle'.. are there plans to support it with gql.tada?

https://graffle.js.org/

**Comment by kitten** [comment]
There's not really much for us to do here.
If they support the `TypedDocumentNode`s "standard" TypeScript typing it'll just work
